src/k8s-engine/k3sHelper.ts

```diff
@@ -20,6 +20,9 @@ import DownloadProgressListener from '@/utils/DownloadProgressListener';
 import safeRename from '@/utils/safeRename';
 import paths from '@/utils/paths';
 import * as K8s from '@/k8s-engine/k8s';
+// TODO: Replace with the k8s version after kubernetes-client/javascript/pull/748 lands
+// const k8s = require('@kubernetes/client-node');
+const findHomeDir = require('@/config/findHomeDir');
```

Comment:
Please use `import` in TypeScript files.

Reply:
noted